### PR TITLE
API-398: Only view logged user jobs in job last executions view

### PR DESCRIPTION
**Pull request by momoss**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added Behats                      | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: features/job-execution/last-execution/export.feature [hunk 1]
```diff
@@ -0,0 +1,27 @@
+@javascript
+Feature: Display only logged user jobs execution in last executions job view
+  In order to have an overview of last job execution
```
Comment (ahocquard) [inline features/job-execution/last-execution/export.feature]: `last job executions`? (plural)

File: features/job-execution/last-execution/import.feature [hunk 2]
```diff
@@ -0,0 +1,33 @@
+@javascript
+Feature: Display only logged user jobs execution in last executions job view
```
Comment (ahocquard) [inline features/job-execution/last-execution/import.feature]: `logged user's job executions`

File: features/job-execution/last-execution/export.feature [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    And I wait for the "csv_footwear_product_export" job to finish
+    And I logout
+    And I am logged in as "admin"
+    And I am on the "csv_footwear_product_export" export job page
+    And I launch the export job
+    And I wait for the "csv_footwear_product_export" job to finish
+    And I am on the job tracker page
+
+  Scenario: Only view last executions of user
+    And I am on the exports grid
```
Comment (anaelChardan) [inline features/job-execution/last-execution/export.feature]: Given

File: features/job-execution/last-execution/export.feature [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    And I am on the "csv_footwear_product_export" export job page
```
Comment (anaelChardan) [inline features/job-execution/last-execution/export.feature]: When

File: features/job-execution/last-execution/export.feature (hunk 3 above)
Comment (anaelChardan) [reply features/job-execution/last-execution/export.feature]: This step is useless no?

File: features/job-execution/last-execution/import.feature [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  In order to have an overview of last job execution
```
Comment (anaelChardan) [inline features/job-execution/last-execution/import.feature]: executions

File: features/job-execution/last-execution/import.feature [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    And I wait for the "csv_footwear_attribute_import" job to finish
+    And I logout
+    And I am logged in as "admin"
+    And I am on the "csv_footwear_attribute_import" export job page
+    And I launch the import job
+    And I wait for the "csv_footwear_attribute_import" job to finish
+    And I am on the job tracker page
+
+  Scenario: Only view last executions of user
+    And I am on the imports grid
```
Comment (anaelChardan) [inline features/job-execution/last-execution/import.feature]: This step is useless no?

**Pull request merged**